types/react-transition-group/react-transition-group-tests.tsx

```diff
@@ -17,6 +17,7 @@ const Test: React.StatelessComponent = () => {
         <TransitionGroup
```

Comment:
Then you should be writing `import TransitionGroup = require('...')`;

There is **absolutely no point** to writing code that pretends to be ES6 but actually can never work when run as ES6. See https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this